mysql-test/suite/clone/include/ddl_common.inc

```diff
@@ -0,0 +1,312 @@
+## Common file for controlled DDL execution with concurrent Clone
+##
+## 1. ddl_op         : DDL operation - CREATE TABLE,  DROP TABLE,  ALTER TABLE ...
+## 2. ddl_stmt       : DDL statement text following table name
+## 3. ddl_post       : Any other follow up clause like partition
+## 4. ddl_table_extn : Second table name extension - new [t1_new, t2_new ...]
+## 5. dml_stmt1      : DML to be executed after DDL during stage-1 FILE COPY
+## 6. dml_stmt11     : DML to be executed after DDL before sending DDL metadata in FILE COPY
+## 7. dml_stmt2      : DML to be executed after DDL during stage-1 PAGE COPY
+## 8. dml_stmt21     : DML to be executed after DDL before sending DDL metadata in PAGE COPY
+## 9. ddl_skip_check : Skip checking data. Used tables are dropped
+##10. ddl_encryption : restart server with keyring plugin
+--source include/big_test.inc
+--source include/not_parallel.inc
+--source include/have_component_keyring_file.inc
+
+--source include/count_sessions.inc
+
+## Get rid of the binary logs from previous run.
+RESET BINARY LOGS AND GTIDS;
+
+--let $CLONE_DATADIR = $MYSQL_TMP_DIR/data_new
+--let $MYSQLD_DATADIR= `select @@datadir;`
+
+if ($remote_clone) {
+  SET GLOBAL DEBUG = '+d,remote_release_clone_file_pin';
+}
+
+if (!$remote_clone) {
+  SET GLOBAL DEBUG = '+d,local_release_clone_file_pin';
+}
```

Comment:
`DBUG_EXECUTE_IF` for both these instrumentations exist in the source code, but they are not covered by any other than this one (which apparently is not being included in any test).